src/contexts/LoginContext.tsx

```diff
@@ -33,45 +34,29 @@ const useLoginContext = (): LoginContextProps => {
 const LoginProvider = ({
   children,
 }: PropsWithChildren<Record<string, never>>): JSX.Element => {
-  const [storedUserId, setStoredUserId, removeStoredUserId] = useLocalStorage(
-    LOCAL_STORAGE_KEYS.GithubId,
-    "Unknown user"
-  )
-  const [
-    storedUser,
-    setStoredUser,
-    removeStoredUser,
-  ] = useLocalStorage(LOCAL_STORAGE_KEYS.User, { email: "", contactNumber: "" })
-  const [
-    storedUserEmail,
-    setStoredUserEmail,
-    removeStoredUserEmail,
-  ] = useLocalStorage(LOCAL_STORAGE_KEYS.Email, "Unknown email")
-
   const [, , removeSites] = useLocalStorage(
     LOCAL_STORAGE_KEYS.SitesIsPrivate,
     false
   )
+  const [storedUserId, setStoredUserId] = useState("")
+  const [storedUserContact, setStoredUserContact] = useState("")
+  const [storedUserEmail, setStoredUserEmail] = useState("")
```

Comment:
do we need to update e2e where we set session defaults etc to come from local storage?

Reply:
Our e2e test settings to bypass the normal checks are on the backend, and localStorage isn't used for anything on the frontend I believe!